ros_ws/src/crazyswarm/src/crazyswarm_server.cpp

```diff
@@ -1682,7 +1562,13 @@ class CrazyflieServer
 
         // Get the unlabeled markers and create point cloud
         if (!useMotionCaptureObjectTracking) {
-          mocap->getPointCloud(markers);
+          // ToDO: switch to pointcloud2 to avoid a copy here
```

Comment:
Might be useful to add details to this TODO in case a contributor comes along. Is this because PCL 2 can make a non-owning point cloud object from a block of memory, but PCL 1 can't? Maybe also clarify that the TODO is for `libobjecttracker` as well as crazyswarm